src/helpers/localStorageHelpers.js

```diff
@@ -81,17 +81,16 @@ export const removeFavoriteCategory = (category) => {
   return true;
 };
 
-export const getDrafts = () => {
-  return store.get('drafts') || {};
-};
+export const getLayout = () =>
+  store.get('layout') || 'card';
```

Comment:
Nested path (app.locale, app.layout) are not working using persistState.
![image](https://cloud.githubusercontent.com/assets/16245250/21500329/347bf0e4-cc3e-11e6-8c58-5ea0355b568c.png)
I'm not sure it's relevant to have a root state object for every features, i prefer having all the settings into 1 state (app).